### DISCUSSION TITLE
Nether /trapped alternative to "Sorry, unable to find a safe location to teleport you to."

### DISCUSSION THREAD

**Discussion by NullCase** [GriefPrevention/GriefPrevention#470] [General]
In The Nether, players trapped in land claims are often unable to get free.

I wish to select an alternate teleport location which is used when a location cannot be found.

What do you think?

**Comment by RoboMWM** [comment]
Because the vanilla nether has a ceiling and stuff, the simple check for a safe spot won't work here. BigScary likely never bothered with this since he defaults to leaving the nether and end unclaimed.

**Comment by bigpresh** [comment]
FWIW, on my server, you can just use `/spawn` to go back to the main overworld spawn at any time, so you're never properly trapped - possibly a workaround, although not perfect - particularly if you don't allow teleporting back to the spawn point.

**Comment by lerokko** [comment]
A possible solution for this could be a `/setclaimspawn x y z` command. You could define hard coordinates (for the claim you stand in) to which to teleport if someone uses /trapped.

**Comment by RoboMWM** [comment]
I believe GriefPrevention flags allows a player to specify an exit location for /trapped

**Comment by NullCase** [comment]
Seems like something to look into.

**Comment by ShaneBeee** [comment]
> I believe GriefPrevention flags allows a player to specify an exit location for /trapped

Just to clarify, yes this is a [**FLAG**](https://github.com/ShaneBeee/GriefPreventionFlags/wiki/Flags#trappeddestination) in GP Flags

**Comment by NullCase** [comment]
Hi @RoboMWM Funding is available for this issue. Because I had a fresh reminder today.

I don't know what the best solution is. What do you think about a simple design goal where /trapped somehow functions in the Nether to free people?

**Comment by RoboMWM** [comment]
Well, I guess one way to come up with a solution is... how would you find a safe place for the player to teleport to? And then pay attention to the steps you take.

**Comment by ShaneBeee** [comment]
What about something like picking the lesser corner of the claim, taking the block south of it (or whatever direction from the claim that would be outside of the claim)... looping blocks up until you find 2 air blocks to TP the player to?
